histolab/util.py

```diff
@@ -351,6 +355,6 @@ def method_dispatch(func: Callable[..., Any]) -> Callable[..., Any]:
     def wrapper(*args, **kw):
         return dispatcher.dispatch(args[1].__class__)(*args, **kw)
 
-    wrapper.register = dispatcher.register
+    setattr(wrapper, "register", dispatcher.register)
```

Comment:
curiosity: why this?

Reply:
doing `wrapper.register = xxx` throws a mypy error because `register` doesn't exist on wrapper so as a workaround we use setattr 😊